src/Plugin.php

```diff
@@ -107,6 +107,9 @@ public static function init() {
     add_filter('script_loader_tag', __NAMESPACE__ . '\Asset::loader_tag', 10, 3);
     add_filter('style_loader_tag', __NAMESPACE__ . '\Asset::loader_tag', 10, 3);
 
+    // Filter whether to kill elasticpress sync during imports.
+    add_filter('ep_sync_indexable_kill', '__return_false', 1, 2);
```

Comment:
Why accepting two arguments?

```suggestion
    add_filter('ep_sync_indexable_kill', '__return_false', 1);
```

Reply:
It passes 2 arguments right? http://10up.github.io/ElasticPress/ep_sync_indexable_kill.html

Reply:
Yes, but your callback is accepting zero, so it is unnecessary to register a specific amount.

Reply:
Thanks for clarifying